indico/modules/events/registration/models/forms.py

```diff
@@ -525,6 +533,11 @@ def get_ticket_template(self):
         from indico.modules.designer.util import get_default_ticket_on_category
         return self.ticket_template or get_default_ticket_on_category(self.event.category)
 
+    @property
+    def is_google_wallet_configured(self):
```

Comment:
Wouldn't it make more sense to also check if it's enabled in here?

```python
    @property
    def is_google_wallet_configured(self):
        return (
            config.ENABLE_GOOGLE_WALLET and
            self.ticket_google_wallet_enabled 
            GoogleWalletManager.get_google_wallet_settings(self.event.category) is not None and
        )
```

Or have `is_google_wallet_available` (checks config+category and ``is_google_wallet_enabled` (checks available + enabled)

Reply:
self.ticket_google_wallet_enabled is only a switch to be displayed at Category form settings, it does not contain anything: when the switch is enabled, you HAVE TO submit google wallet configuration, and those are saved an retrieved by GoogleWalletManager.get_google_wallet_settings(self.event.category) that search for inherited confs.
So having the google_wallet_settings means that you have a potential google wallet connection available, and if config.ENABLE_GOOGLE_WALLET is True THEN is_google_wallet_configured returns True and you have green light to display the link.

Reply:
At the moment, I do not see the case of having the Google Wallet enabled for a parent category and explicitly DISABLED for a subcategory branch... in that case I should use the switch in a different way, to stop inheriting.
Let's keep this feature for the future, if requested?

Reply:
Just adding my two cents here. I see three useful flags:
- `is_google_wallet_enabled`: Whether the regform has Google Wallet enabled.
- `is_google_wallet_configured`: Checks if Google Wallet is configured in `config` and category.
- `is_google_wallet_available`: Checks for `is_google_wallet_enabled` and `is_google_wallet_configured`.

Reply:
I think a "stop inheriting" option is overkill - we do not support this elsewhere (ACLs aside) either at the moment either.

Reply:
Ok, I think in my previous comment I mixed ticket_google_wallet_enabled, that is a REGISTRATION FORM property, with another ticket_google_wallet_enabled that is just a display switch in TicketsForm. My bad.
I'll do the requested changes.